Tools-Override/tests.targets

```diff
@@ -55,11 +60,72 @@
 
     <TestCommandLine Condition="'$(Performance)'!='true'">$(TestProgram) $(TestArguments) {XunitTraitOptions}</TestCommandLine>
   </PropertyGroup>
+  
+  <!-- This will generate a version of the dotnet cli that runs with the live
+       build of corefx using the other components gathered from the stable 
+       version consumed from the Tools directory 
+  -->
+  <Target Name="ConstructDotNetExe" Condition="!Exists('$(BinDir)dotnetcli')" >
+    <PropertyGroup>
+      <TestDotNetCliPath>$(BinDir)dotnetcli</TestDotNetCliPath>
+    </PropertyGroup>
+    <ItemGroup>
+      <DotNetCliFiles Include="$(DotnetCliPath)\*.*" />
+      <DotNetCliSharedFramework Include="$(DotnetCliPath)\shared\**\*.*" />
+      <DotNetCliSharedFrameworkHost Include="$(DotnetCliPath)\host\**\*.*" />
+      <DotNetCliSharedFrameworkSdk Include="$(DotnetCliPath)\sdk\**\*.*" />
+      <DotNetCliSharedFrameworkSdkRuntimeConfig Include="$(TestDotNetCliPath)\sdk\**\dotnet.runtimeconfig.json" />
+      <LiveDotNetCliComponents Include="$(RuntimePath)\*.*" Exclude="$(RuntimePath)\xunit.console.netcore.exe" />
+    </ItemGroup>
+  
+    <Copy SourceFiles="@(DotNetCliFiles)" 
+          DestinationFiles="@(DotNetCliFiles -> '$(TestDotNetCliPath)\%(RecursiveDir)%(Filename)%(Extension)')" />
+    <Copy SourceFiles="@(DotNetCliSharedFrameworkHost)" 
+          DestinationFiles="@(DotNetCliSharedFrameworkHost -> '$(TestDotNetCliPath)\host\%(RecursiveDir)%(Filename)%(Extension)')" />
+    <Copy SourceFiles="@(DotNetCliSharedFrameworkSdk)" 
+          DestinationFiles="@(DotNetCliSharedFrameworkSdk -> '$(TestDotNetCliPath)\sdk\%(RecursiveDir)%(Filename)%(Extension)')" />
+    <ItemGroup>     
+      <DotNetCliSharedFrameworkSdkRuntimeConfig Include="$(TestDotNetCliPath)\sdk\**\dotnet.runtimeconfig.json" />
+    </ItemGroup>
+    <ReadLinesFromFile File="@(DotNetCliSharedFrameworkSdkRuntimeConfig)" >  
+        <Output TaskParameter="Lines"  
+                ItemName="LinesFromDotNetCliSharedFrameworkSdkRuntimeConfig"/> 
+    </ReadLinesFromFile>
+    <ItemGroup>
+       <LinesToWriteDotNetCliSharedFrameworkSdkRuntimeConfig Include="$([System.String]::Copy('%(LinesFromDotNetCliSharedFrameworkSdkRuntimeConfig.Identity)').Replace('1.0.0','9.9.9'))" />
+       <LinesToWriteDotNetCliSharedFrameworkSdkRuntimeConfig Include="};}" />
```

Comment:
Since this method copies over everything from the RuntimePath this will definitely include any new libraries that have been added and includes them in the deps.json so that they will be a part of the shared framework.